src/os/file/file.js

```diff
@@ -313,27 +323,52 @@ os.file.createFromContent = function(fileName, url, originalFile, content) {
 
 
 /**
- * Creates a local:// url used by file storage
- * @param {string} fileName The file name to use in generating the url
+ * Creates a `file://` url to reference files on the file system.
+ * @param {string} path The path to the file.
+ * @return {string}
+ */
+os.file.getFileUrl = function(path) {
+  return os.file.FileScheme.FILE + '://' + path;
+};
+
+
+/**
+ * Creates a `local://` url used by file storage.
+ * @param {string} fileName The file name to use in generating the url.
  * @return {string}
  */
 os.file.getLocalUrl = function(fileName) {
-  return os.file.File.URL_SCHEME + '://' + goog.crypt.base64.encodeString(fileName);
+  return os.file.FileScheme.LOCAL + '://' + goog.crypt.base64.encodeString(fileName);
```

Comment:
I know this doesn't have anything to do with your change, but was there a reason that we were using `goog.crypt.base64.encodeString()` over the native `window.btoa()`?